quipucords/scanner/vcenter/tests_vc_connect.py

```diff
@@ -109,7 +109,7 @@ def test_connect(self):
             with patch('scanner.vcenter.connect.get_vm_names',
                        return_value=['vm1', 'vm2']) as mock_names:
                 vm_names = self.runner.connect()
-                self.assertEqual(vm_names, set(['vm1', 'vm2']))
```

Comment:
Do you think it would be useful to update the test to make `get_vm_names` return a list with at least one duplicated name and ensure the list will be returned as expected?

Reply:
👍 Done